src/excited_workflow/source_datasets/land_cover.py

```diff
@@ -22,7 +22,41 @@ def _cftime_to_datetime(data: xr.DataArray) -> np.ndarray:
     return np.array([np.datetime64(el) for el in data.to_numpy()])
 
 
-def dims_check(dat: xr.Dataset, target: xr.Dataset, var: str) -> bool:
+def regrid(
+    file: Path,
+    target_grid: xr.Dataset,
+    variables: list[str] | None = None,
+) -> xr.Dataset:
+    """Regrids one file to target dataset.
+
+    Args:
+        file: file to regrid
+        variables: List of variable names which should be downloaded.
+        target_grid: Grid to which the data should be regridded to.
+
+    Returns:
+        Regridded Dataset.
+    """
+    ds = xr.open_dataset(file, chunks={"lat": 2000, "lon": 2000})
+
+    # Set time to middle of bounds.
+    time_coords = _cftime_to_datetime(ds["time_bounds"].mean(dim="bounds"))
+    ds = ds.drop("time_bounds")
+    ds["time"] = time_coords
+
+    ds = ds[["lccs_class"]]  # Only take the class variable.
```

Comment:
The variables argument is not used here then, right? The variables are already limited to only lccs_class

Reply:
I think I just took this from the demo but you're right doesn't apply here.